gui/app/services/boardService.js

```diff
@@ -116,6 +116,15 @@
         return scenes;
     }
     
+    service.getControlIdsForSelectedBoard = function (){
+        var board = service.getLastUsedBoard();
+        var controls = [];
+        if (board != null) {
+          controls = _.keys(board.controls);
```

Comment:
Per Probable's recommendation, I am going to actually remove underscore.js since the newer version of Javascript supports most of the functions that underscore. So anything going forward should use native implementations.

In this case, instead of `_.keys()` you can do `Object.keys(board.controls)`